lib/tasks/bootstrap.rake

```diff
@@ -0,0 +1,89 @@
+namespace :bootstrap do
+  desc 'Retrieve dependencies needed for project to run'
+  task :dependencies do
+    unless npm_installed?
+      puts "Project requires npm to run: 'brew install npm' "
+      puts "Quitting .."
+      next
+    end
+
+    unless pg_installed?
+      puts "Project requires Postgresql to run, please install it"
+      puts "Quitting .."
+      next
+    end
+
+    sh 'gem install bundler' unless bundler_installed?
+
+    sh 'bundle install'
+    sh 'npm install -g lineman' unless lineman_installed?
+    sh 'npm install -g bower' unless bower_installed?
+    sh 'cd lineman && bower install'
+  end
+
+  desc "Create database.yml file"
+  # don't add :environment here, you can't load Rails without database file
+  task :config_files do
+    unless File.exists?(File.join(Dir.pwd, "config", "database.yml"))
+      source = File.join(Dir.pwd, "config", "database.example.yml")
+      target = File.join(Dir.pwd, "config", "database.yml")
+      FileUtils.cp_r source, target
+      puts "Database.yml file created"
+    else
+      puts "Database.yml file already exists"
+    end
+  end
+
+  desc 'Create user (optional arguments email)'
+  task :create_user, [:email, :password] => :environment do |t, args|
+    args.with_defaults[email: 'default@loomio.com', password: 'passcode1']
+    if User.find_by(email: args[:email]).empty?
```

Comment:
Running `empty?` on `nil` is going to throw an error, so this is no good. Either switch up the logic to use present, or use `nil?` here instead?